nservicebus/hosting/nservicebus-host/installation.md

```diff
@@ -20,83 +32,103 @@ NServiceBus.Host.exe [/install [/serviceName]
 [/sideBySide]
 [/startManually]
 [/username]
-[/password]]
-[/uninstall [/serviceName]
-[/sidebyside]
-[/instance:Instance Name ]
+[/password]
 ```
 
-To retrieve this list, run the following at the command line:
+Here is an example of the `/install` command line:
 
 ```dos
-NServiceBus.Host.exe /?
+NServiceBus.Host.exe /install 
+/serviceName:"MyPublisher"
+/displayName:"My Publisher Service"
+/description:"Service for publishing event messages"
+/endpointConfigurationType:"QualifiedNameSpace.EndpointConfigType, AssemblyName"
+/username:"corp\serviceuser"
+/password:"p@ssw0rd!" NServiceBus.Production
```

Comment:
@timbussmann - In older versions of NServiceBus (maybe version 3 and below), you had to specify the profile. In the current versions, NServiceBus.Production is the default profile and therefore I don't think needs to be specified.